snapshot-replicator/functions/remove_snapshots.py

```diff
@@ -10,6 +11,15 @@
 print('Loading function')
 
 def lambda_handler(event, context):
+    source = boto3.client('rds', region_name=source_region)
+    for instance in instances.split(','):
+        for snapshot in target.describe_db_snapshots(DBInstanceIdentifier=instance, MaxRecords=50)['DBSnapshots']:
```

Comment:
you'll need to paginate the results here, otherwise the results are not complete. It was precisely this that caused the redis backups to fail 😅

Reply:
also, can you put this into a python function so it can be reused for both regions?

Reply:
oh I didn't know the lambda function itself had issues (yet to check redis). Ok, I'll check this further